### PR TITLE
parrot gif added

### PR DESCRIPTION
**Pull request by bolajiscott**
new parrots gif added up

### CODE REVIEW

File: parrots.yaml [hunk 1]
```diff
@@ -466,4 +466,10 @@
   tip: Looks kinda sus
 - gif: chefparrot.gif
   name: Chef parrot
-  tip: Cooking parrot
\ No newline at end of file
+  tip: Cooking parrot
+  gif: appleparrots.gif
+  name: apple parrot
+  tip: contesting
```
Comment (jmhobbs) [inline parrots.yaml:472]: Tip probably isn't needed on this one.

File: parrots.yaml [hunk 2]
```diff
@@ -466,4 +466,10 @@
   tip: Looks kinda sus
 - gif: chefparrot.gif
   name: Chef parrot
-  tip: Cooking parrot
\ No newline at end of file
+  tip: Cooking parrot
+  gif: appleparrots.gif
+  name: apple parrot
```
Comment (jmhobbs) [inline parrots.yaml:471]: No offense, but that really looks like a pear to me.  Can we change the gif to `pear-parrots.gif` and name to `Pear Parrots`?  Is there a story behind this gif?

File: parrots.yaml [hunk 3]
```diff
@@ -466,4 +466,10 @@
...
+  tip: contesting
+
+
+
```
Comment (jmhobbs) [inline parrots.yaml:475]: Trim out these empty lines please 😄

**Review by jmhobbs — CHANGES_REQUESTED** [review]

**Comment by bolajiscott** [comment]
changes made

    On Friday, October 30, 2020, 11:05:11 PM GMT+1, John Hobbs <notifications@github.com> wrote:  
 
 


@jmhobbs requested changes on this pull request.

In parrots.yaml:
> \ No newline at end of file
+  tip: Cooking parrot
+  gif: appleparrots.gif
+  name: apple parrot
+  tip: contesting

Tip probably isn't needed on this one.

In parrots.yaml:
> \ No newline at end of file
+  tip: Cooking parrot
+  gif: appleparrots.gif
+  name: apple parrot

No offense, but that really looks like a pear to me. Can we change the gif to pear-parrots.gif and name to Pear Parrots? Is there a story behind this gif?

In parrots.yaml:
> \ No newline at end of file
+  tip: Cooking parrot
+  gif: appleparrots.gif
+  name: apple parrot
+  tip: contesting
+
+
+  

Trim out these empty lines please 😄

File: parrots.yaml (hunk 2 above)
Comment (bolajiscott) [reply parrots.yaml]: lol its been updated to pear. Accept PR

File: parrots.yaml (hunk 3 above)
Comment (bolajiscott) [reply parrots.yaml:472]: done

**Review by jmhobbs — APPROVED** [review]

**Comment by jmhobbs** [comment]
Thanks!

**Pull request merged**